Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -98,6 +99,11 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode
   vtkGetMacro(Visibility, int);
   vtkSetMacro(Visibility, int);
 
+  /// Get interaction node.
+  /// If no node has been explicitly set, it the singleton interaction node.
```

Comment:
Here it is:
```
If no node has been explicitly set using SetInteractionNode() or SetInteractionNodeID(), return
the singleton interaction node.

The singleton interaction node is considered to be the default interaction node. Associating
a specific interaction node to one or a multiple views allows to control the interaction mode
associated with these views independently.

Since by default, the interaction node is a singleton, a new interaction node may be
created doing the following:

  vtkNew<vtkMRMLInteractionNode> interactionNode;
  interactionNode->SetSingletonOff()
  this->GetMRMLScene()->AddNode(interactionNode.GetPointer());

```